test/integration/admin/api/backend_apis_controller_test.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::API::AccountsControllerTest < ActionDispatch::IntegrationTest
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+    host! @provider.admin_domain
+    @access_token_value = FactoryBot.create(:access_token, owner: @provider.admin_users.first!, scopes: %w[account_management], permission: 'rw').value
+    Logic::RollingUpdates.stubs(enabled?: true)
+    Logic::RollingUpdates::Features::ApiAsProduct.any_instance.stubs(:enabled?).returns(true)
+    @backend_api = FactoryBot.create(:backend_api, account: provider)
```

Comment:
This is been created for each tests (though create for example does not need it)